### PR TITLE
fix: [PL-22704]: Disable Harness SM fixes

### PR DESCRIPTION
**Pull request by meenakshi-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by meenakshi-harness** [comment]
retrigger titlecheck

File: src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx [hunk 1]
```diff
@@ -79,28 +81,44 @@ const ConnectorSettings: React.FC<ConnectorSettingProps> = props => {
...
     <Layout.Horizontal flex={{ justifyContent: 'space-between' }} margin={{ top: 'small', bottom: 'small' }}>
-      <Checkbox
-        labelElement={
-          <div>
-            <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_900}>
-              {getString('common.accountSetting.connector.disableBISMHeading')}
+      <Popover
+        interactionKind={PopoverInteractionKind.HOVER}
+        boundary="viewport"
+        disabled={!disabled}
```
Comment (mayankVermaHarness) [inline src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx:87]: `disabled={!disabled}`
Is this expected?

Comment (meenakshi-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx:87]: Yes

**Review by mayankVermaHarness — DISMISSED** [review]

File: src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx [hunk 2]
```diff
@@ -79,28 +81,44 @@ const ConnectorSettings: React.FC<ConnectorSettingProps> = props => {
...
+        content={
+          loading || loadingSecretsManagers ? (
+            <Text padding="medium" font={{ variation: FontVariation.SMALL }} color={Color.GREY_800}>
+              {getString('common.accountSetting.connector.loading')}
             </Text>
-            <Text font={{ variation: FontVariation.SMALL }} color={Color.GREY_600}>
-              {getString('common.accountSetting.connector.disableBISMSubHeading')}
+          ) : (
+            <Text padding="medium" font={{ variation: FontVariation.SMALL }} color={Color.GREY_800}>
+              {getString('common.accountSetting.connector.disabledState')}
```
Comment (avinashmadhwani02) [inline src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx:95]: You can render only text based on condition.

loading || loadingSecretsManagers ? {getString('common.accountSetting.connector.loading')} : {getString('common.accountSetting.connector.disabledState')}

Comment (meenakshi-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx:95]: I need font variation and color as well.

**Review by reetika04 — APPROVED** [review]

File: src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx (hunk 2 above)
Comment (meenakshi-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx]: fixed

[the remark this answers is not in the document]
Comment (meenakshi-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx]: Will do in master merges

[the remark this answers is not in the document]
Comment (avinashmadhwani02) [reply src/modules/33-auth-settings/pages/AccountOverview/views/Settings/connector/ConnectorSettings.tsx]: Yeah we should put all the tooltip texts at once place

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**